app/blueprints/assessments/templates/activity_trail.html

```diff
@@ -0,0 +1,168 @@
+{% extends "base.html" %}
+{% from "macros/theme.html" import theme %}
+{% from "govuk_frontend_jinja/components/label/macro.html" import govukLabel %}
+{%- from "govuk_frontend_jinja/components/table/macro.html" import govukTable -%}
+{%- from "govuk_frontend_jinja/components/fieldset/macro.html" import govukFieldset -%}
+{%- from "govuk_frontend_jinja/components/textarea/macro.html" import govukTextarea -%}
+{%- from "govuk_frontend_jinja/components/checkboxes/macro.html" import govukCheckboxes -%}
+{%- from "govuk_frontend_jinja/components/radios/macro.html" import govukRadios %}
+{%- from 'govuk_frontend_jinja/components/back-link/macro.html' import govukBackLink -%}
+{% from "macros/logout_partial.html" import logout_partial %}
+{% from "macros/banner_summary.html" import banner_summary %}
+{% from "macros/assessment_flag.html" import assessment_flag as assessment_flagged %}
+{% from "macros/assessment_flag.html" import assessment_resolve as assessment_resolved %}
+{% from "macros/assessment_flag.html" import assessment_stop as assessment_stopped %}
+{% from "macros/comments.html" import comments %}
+{% from "macros/scores_justification.html" import score as scores %}
+{% from "macros/tags_table.html" import tags %}
+
+
+{% block header %}
```

Comment:
Yes we need the header to display  the banner summary and other information unless I am missing something? 🤔

Reply:
Yea, but this looks copy pasted, rather than using the `header.html` like this: https://github.com/communitiesuk/funding-service-design-assessment/blob/e631c389272ded1e20889f84b07b68137b944602/app/blueprints/tagging/templates/change_tags.html#L4-L6

Reply:
Ah! That’s exactly the answer I was looking for, nice 👍